### ISSUE TITLE
How can we use webhooks to call the API

### ISSUE THREAD

**Issue by calyweb** [mirakl/sdk-php-shop#31]
Hi,
Does this SDK contains webhooks to call the API, i mean for example call the API when a new order is placed on Mirakl ? And not use a cron task to check every 5 minutes if there is a new order
Many thanks for your feedback !

**Comment by TriJoe** [comment]
I don't believe it supports webhooks. I use cron to request orders from the last pulled order date.

**Comment by igitur** [comment]
So why is there a reference to order webhooks on https://help.mirakl.net/bundle/sellers/page/topics/Mirakl/concepts/Order/product-title-localization.htm ?

**Comment by TriJoe** [comment]
This may be new. I've yet to see anywhere to setup webhooks on Mirakl. Now it could be that the access permissions for my account has been limited, but still, I am unable to find any significant documentation on them either.
